src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Serialization.cs

```diff
@@ -24,22 +24,17 @@ internal partial class SymbolTreeInfo : IObjectWritable
         /// info can't be loaded, it will be created (and persisted if possible).
         /// </summary>
         private static Task<SymbolTreeInfo> LoadOrCreateSourceSymbolTreeInfoAsync(
-            Solution solution,
-            IAssemblySymbol assembly,
-            Checksum checksum,
-            string filePath,
-            bool loadOnly,
-            CancellationToken cancellationToken)
+            Project project, Checksum checksum, bool loadOnly, CancellationToken cancellationToken)
         {
             return LoadOrCreateAsync(
-                solution,
+                project.Solution,
                 checksum,
-                filePath,
+                project.FilePath,
```

Comment:
What is this used for internally?  It turns out that a project's filepath is not enough to uniquely distinguish it.